Posts/2021/03/ScriptingGuy.0007.folderexists.md

```diff
@@ -0,0 +1,40 @@
+---
+post_title: Determine if a folder exists
+username: fabian.baumanis@gmx.de
```

Comment:
@termdew Before we can merge this, you need to create your WordPress account. I don't see an account for you in WordPress. For instructions, see https://github.com/PowerShell/Community-Blog/wiki/Becoming-a-contributor